c++/Makefile.am

```diff
@@ -274,6 +275,7 @@ libkj_la_SOURCES=                                              \
   src/kj/string.c++                                            \
   src/kj/string-tree.c++                                       \
   src/kj/source-location.c++                                   \
+  src/kj/glob-filter.c++
```

Comment:
You're missing the trailing backslash on this line.

Reply:
doh! lol sigh.